src/sickle.ts

```diff
@@ -263,6 +266,29 @@ class Annotator {
     this.emit(': void;\n');
   }
 
+  private visitEnum(node: ts.EnumDeclaration) {
+    this.emit('/** @typedef {number} */\n');
```

Comment:
One more thing: should this be `@typedef` or `@enum`? I'd prefer `@enum` if Closure swallows a non-initialized enum.